src/main/java/com/shopify/ShopifySdk.java

```diff
@@ -451,6 +455,38 @@ public int getProductCount() {
 		return count.getCount();
 	}
 
+	public List<ShopifyCustomCollection> getCustomCollections(final int page, final int pageSize) {
```

Comment:
This call will no longer work after 2019/07/20. I'm not sure if this is something we want to address right now @ryankazokas but just wanted to bring to light that this is another call we'll need to fix along with #20 

 https://help.shopify.com/en/api/reference/products/customcollection?api[version]=2019-07

If we were to address this right now, we would need to add an optional `API version` field to the `ShopifySDK Builder`

Reply:
@rjdavis3 i don't believe we should handle that at the moment as this will take much more work.

I'm pretty sure there will be breaking changes because as of 2019-10 pagination isn't supported anywhere so these methods will have to change or the way we call them, we'll have to use the version number to know if they can be called.

I might have a few hours over the weekend I can possibly look at handling that and the version number.

Reply:
@ryankazokas I agree with waiting just wanted to note. I'll resolve this conversation. Let's not worry about changing anything until we are ready to address #20